standard.md

```diff
@@ -6,17 +6,17 @@
-**4. Platform Independence** | If the project has mandatory dependencies that create more restrictions than the original license, the project(s) must be able to demonstrate independence from the closed component(s) and/or indicate the existence of functional, open alternatives.
-**5. Documentation** | The project must have documentation of the source code, use cases, and/or functional requirements. For content, this should include all relevant/compatible apps, software, or hardware required to access the content, and instructions regarding how to use it. For software projects, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how data was collected, and how it should be interpreted.
-**6. Mechanism for Extracting Data and Content** | If the project has non personally identifiable information (PII) there must be a mechanism for extracting or importing non-PII data and content from the system in a non-proprietary format.
-**7. Adherence to Privacy and Applicable Laws** | The project must state to the best of its knowledge that it complies with relevant privacy laws, and all applicable international and domestic laws.
-**8. Adherence to Standards & Best Practices** | Projects must demonstrate adherence to standards, best practices, and/or principles. For example, [the Principles for Digital Development](https://digitalprinciples.org/principles/).
-**9. Do No Harm By Design** | All projects must demonstrate that they have taken steps to ensure the project anticipates, prevents, and does no harm.
-**9a) Data Privacy & Security** | Projects collecting data must identify the types of data collected and stored. Projects must also demonstrate how they ensure the privacy and security of this data in addition to the steps taken to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9b) Inappropriate & Illegal Content** | Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials in addition to mechanisms for detecting, moderating and removing inappropriate/illegal content.
-**9c) Protection from Harassment** | If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users. 
+**1. Relevance to Sustainable Development Goals** | Digital public goods must be designed and developed to advance the [Sustainable Development Goals (SDGs)](https://sdgs.un.org/goals) and demonstrate such by providing links/documentation that support their relevance. 
```

Comment:
In the past we did not require nexus to SDG's to be demonstrated within the DPG products themselves, but rather by the project community as part of their nomination process or by pointing to pre-existing documentation living outside of the DPG product(s) such as on web sites, wikis, etc.

Reply:
This is stronger to prevent tenuous claims of relevance. Technically, even the open source software this github repo is running on, is advancing the SDG by helping us host a list of DPGs. However, this would make the registry noisy and the same as an Open Source registry. To distinguish all open source project from the select few that qualify as an SDG, we’re making this standard stricter.